src/site/content/en/blog/speed-to-business/index.md

```diff
@@ -0,0 +1,177 @@
+---
+layout: post
+title: Relating Site Speed and Business Metrics
+subhead: Leverage A/B testing to evaluate the impact of site speed on your business metrics
+authors:
+  - msdikla
+  - martinschierle
+  - bpj 
+date: 2020-06-25
+hero: hero.jpg
```

Comment:
How about this for the hero:

![louis-reed-pwcKF7L4-no-unsplash](https://user-images.githubusercontent.com/4713486/86873180-ad17d580-c092-11ea-8a83-1cddb94dfa25.jpg)